meson.build

```diff
@@ -87,6 +87,14 @@ if get_option('win32') != 'false'
   use_win32 = dependency('libva-win32', required: require_win32).found()
   if use_win32
     libva_utils_flags = [ '-DHAVE_VA_WIN32=1', '-DNOMINMAX']
+    dep_dxheaders = dependency('directx-headers', required : false)
```

Comment:
why did ci build work when you had a typo here?

Reply:
Because after not finding the `directxheaders-` package with the typo, it was using the fallback [DirectX-Headers project](https://github.com/microsoft/DirectX-Headers) that was being cloned in the CI actions (and now cloned as a mesa subproject dependency). 

https://mesonbuild.com/Subprojects.html#toggling-between-system-libraries-and-embedded-sources

![image](https://user-images.githubusercontent.com/82116493/197613410-cf901f9e-f7a5-46e0-8e6c-0c1ec2bc6ff3.png)